README.md

```diff
@@ -22,7 +22,7 @@ general technical documentation. In addition, there are project-specific
 Contributing
 ------------
 
-Patches are welcome! Feel free to fork and contribute to this project on
+Your PR (Pull Requests) are welcome! Feel free to fork and contribute to this project on
```

Comment:
I would actually simply change this to read "Your pull requests are welcome!...."